### ISSUE TITLE
[LUIS Authoring] Missing 429 Response Code in the Swagger

### ISSUE THREAD

**Issue by sarangan12** [Azure/azure-rest-api-specs#19052]
This issue has been reported by customer related to LUIS Authoring JS SDK. While making calls with the SDK, the service returns a 429 Response code. This status code refers to Too Many Requests. This response code should have been handled by RetryPolicy. But, the SDK considers it as an error. 

While investigating this issue, I found the response code `429` was never defined in the Swagger which is incorrect. You can find several examples in specs repository where services define 429 as valid response code. (Web, Communication, etc). A service is supposed to define all possible response codes. But, this service does not define 429 code but responds with it. This swagger has to be fixed.

File to be updated: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

**Comment by moabba** [comment]
@heaths as far as I know, the guidance we've been getting is to only include the success status codes and have a "default" status code for all other failures. What's the action we should take here? This issue will apply for Language (and any APIM-backed service) as well.

**Comment by heaths** [comment]
Correct. [Our guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=YOU%20SHOULD%20NOT%20document%20specific%20error%20status%20codes%20in%20your%20OpenAPI/Swagger%20spec%20unless%20the%20%22default%22%20response%20cannot%20properly%20describe%20the%20specific%20error%20response%20(e.g.%20body%20schema%20is%20different).) state specific error codes should not be documented. Any error could happen and callers should be prepared.
